qiita_ware/commands.py

```diff
@@ -70,22 +71,41 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
Okay. In that case, if it isn't possible (or expected) that `generate_send_sequences_cmd()` will except, I recommend:

```python
environ['ASPERA_SCP_PASS'] = qiita_config.ebi_seq_xfer_pass
seqs_cmds = ebi_submission.generate_send_sequences_cmd()
try:
    # place holder for moi submission, see PR notes in #1477
    pass
except:
    LogEntry.create('Fatal', seqs_cmds, info={'ebi_submission': preprocessed_data_id})
environ['ASPERA_SCP_PASS'] = old_ascp_pass
```

This limits the `try` to the single in which we anticipate an exception happening.